sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Table.java

```diff
@@ -242,7 +242,7 @@ public synchronized void close() {
             JsonObject obj = new JsonObject();
             obj.put("rid", rid);
             obj.put("stream", StreamState.CLOSED.getJsonName());
-            writer.writeResponse(obj);
+            writer.writeResponse(obj,false);
```

Comment:
Method calls with multiple passed parameters should have spaces after commas. Same with line 339.